quantum/led_matrix/led_matrix.h

```diff
@@ -43,6 +43,7 @@
 
 #ifndef LED_MATRIX_LED_PROCESS_LIMIT
 #    define LED_MATRIX_LED_PROCESS_LIMIT (LED_MATRIX_LED_COUNT + 4) / 5
+#    define LED_MATRIX_LED_PROCESS_MAX_ITERATIONS ((LED_MATRIX_LED_COUNT + LED_MATRIX_LED_PROCESS_LIMIT - 1) / LED_MATRIX_LED_PROCESS_LIMIT)
 #endif
```

Comment:
Im guessing we do not expect users to have to also specify `LED_MATRIX_LED_PROCESS_MAX_ITERATIONS` when configuring `LED_MATRIX_LED_PROCESS_LIMIT`? As right now the code will fail to compile when `LED_MATRIX_LED_PROCESS_LIMIT` is not defined.
```suggestion
#endif
#define LED_MATRIX_LED_PROCESS_MAX_ITERATIONS ((LED_MATRIX_LED_COUNT + LED_MATRIX_LED_PROCESS_LIMIT - 1) / LED_MATRIX_LED_PROCESS_LIMIT)
```

Reply:
Ah yes it should be internal yep.